schema/schema.sql

```diff
@@ -0,0 +1,66 @@
+DROP TABLE IF EXISTS purchase;
+DROP TABLE IF EXISTS item_genre_assignment;
+DROP TABLE IF EXISTS genre;
+DROP TABLE IF EXISTS item;
```

Comment:
May have to check the ordering of these `DROP TABLE`'s. I think you may have to drop `genre` before `item_genre_assignment` since it contains a foreign key

Reply:
You can add commas to drop them all at once regardless of order

Reply:
Something like
`DROP TABLE IF EXISTS item_genre_assignment, genre, item;`

Reply:
I've just tested this locally, all 8 tables are dropped and created when I run the script so the order seems to be fine.